template/docs/conf.py.jinja

```diff
@@ -134,7 +138,7 @@ html_theme_options = {
     "external_links": [
         {% for link in related_projects.replace(' ', '').split(',')|sort -%}
             {"name": "{{ link }}", "url": "https://scipp.github.io{% if link == 'Scipp' %}{% else %}/{{ link|lower }}{% endif %}"},
-        {% endfor %}],
+    {% endfor %}],
```

Comment:
Intentional?

Reply:
Yes, black changed the corresponding line in the rendered `conf.py`.